src/ray/util/event.cc

```diff
@@ -148,26 +148,51 @@ void EventManager::ClearReporters() { reporter_map_.clear(); }
 ///
 thread_local std::unique_ptr<RayEventContext> RayEventContext::context_ = nullptr;
 
+std::unique_ptr<RayEventContext> RayEventContext::first_instance_ = nullptr;
+
+std::atomic<int> RayEventContext::first_instance_started_setting_(0);
```

Comment:
could this be atomic < bool > instead?

Reply:
‘struct std::atomic<bool>’ has no member named ‘fetch_or’. Do you know what is the best way to do this?

Reply:
‘struct std::atomic<bool>’ has no member named ‘fetch_or’. Do you know what is the best way to do this?